docs/source/working_with_aiida/caching.rst

```diff
@@ -54,7 +101,10 @@ Of course, using caching would be quite tedious if you had to set ``use_cache``
       disabled:
         - aiida.orm.nodes.data.float.Float
 
-This means that caching is enabled for ``TemplatereplacerCalculation`` and ``Str``, and disabled for all other classes. In this example, manually disabling ``aiida.orm.nodes.data.float.Float`` is actually not needed, since the ``default: False`` configuration means that caching is disabled for all classes unless it is manually enabled. Note also that the fully qualified class import name (e.g., ``aiida.orm.nodes.data.str.Str``) must be given, not just the class name (``Str``). This is to avoid accidentally matching classes with the same name. You can get this name by combining the module name and class name, or (usually) from the string representation of the class:
+In this example, caching is enabled for ``TemplatereplacerCalculation`` and ``Str``, and disabled for all other classes.
```

Comment:
I realize now that this may and probably is broken since we separated the node and process. For example adding `aiida_quantumespresso.calculations.pw.PwCalculation` is now a process class and probably won't work as intended. Instead we will probably have to support to use entry point string in the config file, e.g. `aiida.calculations:quantumespresso.pw`

There is already an open issue for this: https://github.com/aiidateam/aiida-core/issues/2990

Reply:
Hm... so do we remove this configuration example entirely?
I realize that the terminology is probably also unclear, since the example includes Data classes but speaks of "caching".
What does it mean to disable "caching" for a data class?
Does it mean its hash is not computed? Will it affect calculations that have this Data type as an input?

Reply:
I agree that caching of a `Data` node, in the way that caching is defined in AiiDA, makes no sense whatsoever. I would be for removing it and maybe adding more explanation to the introduction to stress what caching means and that it _only_ applies to the prevention of running two identical calculations twice.

The practical problem we then face is that the only classes that make sense to add to the configuration do not actually work. This is not acceptable, so I will now start to work on this issue and try to fix it a.s.a.p. You can already update this PR to remove mentioning of Data nodes being cachable and use entry point strings to target specific calc job classes.
Notation should be `aiida.calculations:quantumespresso.pw` for example.

Reply:
Thanks, will do!

Reply:
Done!